suites/pacific/rbd/tier-2_rbd_mirror_snapshot_regression.yaml

```diff
@@ -154,8 +154,12 @@ tests:
       clusters:
         ceph-rbd1:
           config:
-            imagesize: 2G
-            io-total: 200M
+            ec_pool_config:
+              mirrormode: snapshot
+              mode: image
+            rep_pool_config:
+              mirrormode: snapshot
+              mode: image
             snapshot_schedule_level: "pool" #snapshots can be scheduled at pool, image or cluster(default) level
```

Comment:
[Optional] 
Can we remove this comment?
This looks like a usage info and usage infos can be limited to modules ?

Reply:
Done.